config/openshift/csv/kustomization.yaml

```diff
@@ -1,11 +0,0 @@
-# Adds namespace to all resources.
```

Comment:
config/openshift/csv folder can be deleted. as we are not using the csv file from this location

Reply:
This has been deleted